x-pack/plugins/maps/public/actions/data_request_actions.ts

```diff
@@ -435,14 +455,14 @@ export function autoFitToBounds() {
     // Joins are performed on the client.
     // As a result, bounds for join layers must also be performed on the client.
     // Therefore join layers need to fetch data prior to auto fitting bounds.
-    await dispatch(syncDataForAllJoinLayers());
+    await dispatch(syncDataForAllJoinLayers(true));
```

Comment:
Why are you forcing refresh when calling syncDataForAllJoinLayers and syncDataForAllLayers in autoFitToBounds? I would think that force refresh should be false and the layers should only fetch data if the new bounds trigger a refresh.